sdk/cognitivelanguage/Azure.AI.Language.Text/src/Generated/AILanguageTextClientBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure;
+using Azure.AI.Language.Text;
+using Azure.Core.Extensions;
+
+namespace Microsoft.Extensions.Azure
+{
+    /// <summary> Extension methods to add <see cref="TextAnalysisClient"/> to client builder. </summary>
+    public static partial class AILanguageTextClientBuilderExtensions
```

Comment:
Let's add a customization to rename this to "TextAnalysisClientBuilderExtensions". We did this for Conversations.